wallet/concepts/sdk/index.md

```diff
@@ -56,7 +56,7 @@ to MetaMask Mobile using a QR code.
 
 <p align="center">
   <video width="100%" controls>
-    <source src="/sdk-desktop.mp4" type="video/mp4" />
+    <source src="/public-videos/sdk-desktop.mp4" type="video/mp4" />
```

Comment:
These links don't match the folder name you created `public_videos` and so don't render in the preview